examples/access-plugin-minimal/go.mod

```diff
@@ -0,0 +1,64 @@
+module teleport-sheets
+
+go 1.19
+
+require (
+	github.com/gravitational/teleport-plugins v0.0.0-20230317174934-153c31d8b4cf
+	github.com/gravitational/teleport/api v0.0.0-20230317154531-934cde5cb86b
+	github.com/gravitational/trace v1.2.1
+	google.golang.org/api v0.114.0
+	google.golang.org/grpc v1.53.0
+)
+
+require (
```

Comment:
nit: For a "minimal" example, this is a lot more imports than I would expect. I'm guessing a lot of these are coming from `github.com/gravitational/teleport-plugins`? Would it be worthwhile to remove that dependency and just use `github.com/gravitational/teleport/api` directly, in a similar fashion to the workflows example?

Reply:
The `teleport-plugins` imports are:

```
	"github.com/gravitational/teleport-plugins/lib"
	"github.com/gravitational/teleport-plugins/lib/watcherjob"
```

I like the simplicity of the `watcherjob.NewJob` interface, and it looks like we need `lib.NewProcess(ctx)` to run it. 

I know we're also migrating these libraries to `gravitational/teleport`, though, so I'm guessing we'll need to change these imports at some point anyway (as of now, this code compiles successfully for me on a fresh `ubuntu:20.04` container after installing git and go and checking out this branch—not sure when that will stop working).

What would we need to do to reduce the number of imports?

Reply:
> I like the simplicity of the watcherjob.NewJob interface, and it looks like we need lib.NewProcess(ctx) to run it.

Fair enough. Ideally you'd be able to use the `WatcherJob` interface without importing all the other junk, but that's not really your problem.

> I know we're also migrating these libraries to gravitational/teleport, though, so I'm guessing we'll need to change these imports at some point anyway

Actually, it looks like this was already moved into `gravitational/teleport/integrations`. You can probably easily switch the import, but the current code won't stop working as the old teleport-plugin go module will be available indefinitely.

Unfortunately, that means you will be importing even more code that you don't need, as we are back to the first problem listed here - https://github.com/gravitational/teleport/blob/master/rfd/0010-api.md#why.